frontend/src/containers/PaperDetail/PaperDetail.js

```diff
@@ -92,6 +108,7 @@ class PaperDetail extends Component {
                         />
                         {/* FIXME: review-count should reflect this.state.reviewCount */}
                         <h3 id="review-count">{this.state.reviews.length} reviews</h3>
+                        <Button className="review-add" onClick={this.handleClickReviewAddButton}>Add</Button>
```

Comment:
Maybe we need the `disabled` property checking lengths of a title and content of the review.